linkcovery/webui/app.py

```
@@ -0,0 +1,229 @@
+    """Download image and store in cache directory."""
+    try:
+        digest = hashlib.sha256(image_url.encode("utf-8")).hexdigest()
+        suffix = Path(image_url).suffix.lower()
+        if not suffix or len(suffix) > 5:
+            suffix = ".jpg"
+        filename = f"{digest}{suffix}"
+        path = cache_dir / filename
+        if path.exists():
+            return path
+
+        async with AsyncClient(timeout=10, follow_redirects=True, verify=False, http2=True) as client:
```

Ruff (0.15.0)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)